iotaledger/iri
# Update snapshot to 16:00 UTC 20180128

This is the proposed announced snapshot for milestone 337541.

To make this snapshot as smooth, and as safe as possible, we have provided several scripts for the community to independently validate the snapshot and post the results here.
We suggest the community to do so before 20180128 23:00 UTC, when the coordinator and a new IRI version will be published.

# How to Validate
In order to validate this snapshot, we have provided a script for you to run and validate the Snapshot yourself. Obviously, the main assumption is that you have a fully synced node with the latestSolidSubtangleMilestoneIndex at 337541 , and that you're at least a beginner developer.

1. *Get IXI*: Download the IXI module from https://github.com/iotaledger/Snapshot.ixi and place it in your IXI folder. (easiest way is to do: `git clone https://github.com/iotaledger/Snapshot.ixi Snapshot` in your ixi folder - make sure that the folder is called `Snapshot`!).
2. *Get Validation Script*: https://gist.github.com/th0br0/8b92cfe79b89979c67efdc418ff55860 and run `npm i request`
3. *Run and Validate*: Adjust validation script for your node host/port and run with `node 20180128-validation.js` and then post your results.
